docs/aks.md

```diff
@@ -0,0 +1,177 @@
+    ```
+
+    ??? example "Expected output"
+
+        ```text
+        perconaservermongodb.psmdb.percona.com/my-cluster-name created
+        ```
+
+    !!! note
+
+        This deploys default MongoDB cluster configuration, one mongos node and
+        one config server node. Please see [deploy/cr.yaml](https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v{{ release }}/deploy/cr.yaml)
+        and [Custom Resource Options](operator.md#operator-custom-resource-options)
+        for the configuration options. You can clone the repository with all
+        manifests and source code by executing the following command:
+
+        ```bash
+        $ git clone -b v{{ release }} https://github.com/percona/percona-server-mongodb-operator
+        ```
+
+        After editing the needed options, apply your modified `deploy/cr.yaml` file as follows:
+
+        ```bash
+        $ kubectl apply -f deploy/cr.yaml
+        ```
+
+    The creation process may take some time. When the process is over your
+    cluster will obtain the `ready` status. You can check it with the following
+    command:
+
+    ```bash
+    $ kubectl get psmdb
+    ```
+
+    ??? example "Expected output"
+
+        ```text
+        NAME              ENDPOINT                                           STATUS   AGE
+        my-cluster-name   my-cluster-name-mongos.default.svc.cluster.local   ready    5m26s
+        ```
+
+## Verifying the cluster operation
+
+It may take ten minutes to get the cluster started. When `kubectl get psmdb`
+command finally shows you the cluster status as `ready`, you can try to connect
+to the cluster.
+
+{% include 'assets/fragments/connectivity.txt' %}
+
+## Troubleshooting
+
+If `kubectl get psmdb` command doesn't show `ready` status too long, you can 
+check the creation process with the `kubectl get pods` command:
+
+```bash
+$ kubectl get pods
+```
+
+??? example "Expected output"
+
+    --8<-- "./docs/assets/code/kubectl-get-pods-response.txt"
+
+If the command output had shown some errors, you can examine the problematic
+Pod with the `kubectl describe <pod name>` command as follows:
+
+```bash
+$ kubectl describe pod my-cluster-name-rs0-2
+```
+
+Review the detailed information for `Warning` statements and then correct the
+configuration. An example of a warning is as follows:
+
+`Warning  FailedScheduling  68s (x4 over 2m22s)  default-scheduler  0/1 nodes are available: 1 node(s) didn’t match pod affinity/anti-affinity, 1 node(s) didn’t satisfy existing pods anti-affinity rules.`
+
+## Removing the AKS cluster
+
+To delete your cluster, you will need the following data:
+
+* name of your AKS cluster,
+* AWS region in which you have deployed your cluster.
+
+You can clean up the cluster with the `az aks delete` command as follows (with
+real names instead of `<resource group>` and `<cluster name>` placeholders):
+
+```bash
+$ az aks delete --name <cluster name> --resource-group <resource group> --yes --no-wait
+```
+
```

Comment:
we use `--no-wait` flag so we need to add that cluster deletion takes about 10 minutes